docs/section11/Rule11-14.md

```diff
@@ -0,0 +1,37 @@
+# Service_Water_Heating - Rule 11-14  
+**Schema Version:** 0.0.36  
+
+**Mandatory Rule:** FALSE  
+
+**Rule ID:** 11-14  
+
+**Rule Description:** "Where recirculation pumps are used to ensure prompt availability of service water-heating at the end use, the energy consumption of such pumps shall be calculated explicitly."  
+
+**Rule Assertion:** UNDETERMINED / NOT_APPLICABLE
+
+**Appendix G Section Reference:** Table G3.1 #11, baseline column, (f)
+
+**Evaluation Context:** B_RMD - each building
+**Data Lookup:**   
+**Function Call:** 
+- NONE
+
+**Applicability Checks:**
+- check that there is SWH in the proposed model AND that the building has a 24hr operating schedule
+
+## Applicability Checks:
+- look at each building in the RMD: `for building in B_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each space in the building: `for space in P_RMD.building...:`
+        - check to see if there is at least one SWH use associated with the space: `if len(space.service_water_heating_uses) > 0:`
+            - the rule is applicable: `is_applicable = true`
+      
+    
+    ## Rule Assertion:
+    - if the boolean is_applicable is equal to true, the rule is applicable, return UNDETERMINED and a note letting the reviewer know what to check: `if is_applicable: UNDETERMINED raise_message: "This building has service water heating loads.  Where recirculation pumps are used to ensure prompt availability of service water-heating at the end use, the energy consumption of such pumps shall be calculated explicitly."`
+    - otherwise, rule is not applicable: `else: NOT_APPLICABLE`
+
+## Notes:  
+1. is this an appropriate applicability check?  We could check for recirculation pumps in the proposed design, but given that this rule is evaluated just once per building, I think it's ok to have this rule get flagged for all buildings with SWH
```

Comment:
I agree but this seems like something that should be included in the schema in the ServiceWaterHeatingDistributionSystem data table. Maybe recirculation_pump and then a Pump object can be assigned if applicable? Or even a list of pump objects?

Reply:
I like this idea.

Reply:
I think this is a good discussion item for our weekly call.

Reply:
I think we could fully evaluate this if we changed the context to each `ServiceWaterHeatingDistributionSystem`, check whether `ServiceWaterPiping.is_recirculation_loop` is true, and whether the `ServiceWaterPiping` has a pump.